config/database.yml

```diff
@@ -7,14 +7,14 @@ default: &default
 
 development:
   <<: *default
-  database: canvas_rails_template_development
+  database: project_development
```

Comment:
Let's not update these DB names. there is a script in the ./boostrap file that looks for `canvas_rails_template` and replaces it with the project name